Fails to build on OS X 10.10

The library fails to build on OS X 10.10 (public beta 1). I was attempting to install librabbitmq for Python and noticed build failures, traced it back to a failure to build rabbitmq-c on which librabbitmq relies. I've been using Homebrew to manage the installation, the full verbose output is as follows:

```
$ brew install rabbitmq-c --HEAD --verbose
==> Cloning https://github.com/alanxz/rabbitmq-c.git
git --git-dir /Library/Caches/Homebrew/rabbitmq-c--git/.git status -s
Updating /Library/Caches/Homebrew/rabbitmq-c--git
git config remote.origin.url https://github.com/alanxz/rabbitmq-c.git
git config remote.origin.fetch +refs/heads/master:refs/remotes/origin/master
git fetch origin
git checkout -f master
Already on 'master'
Your branch is up-to-date with 'origin/master'.
git reset --hard origin/HEAD
HEAD is now at a2ee005 Bumping version for development.
git submodule update --init --recursive
git reset --hard origin/HEAD
HEAD is now at a2ee005 Bumping version for development.
git checkout-index -a -f --prefix=/private/tmp/rabbitmq-c-IJWd/
git submodule --quiet foreach --recursive git checkout-index -a -f --prefix=/private/tmp/rabbitmq-c-IJWd/${toplevel/\/Library\/Caches\/Homebrew\/rabbitmq-c--git/}/$path/
==> Downloading https://github.com/rabbitmq/rabbitmq-codegen/archive/rabbitmq_v3_3_1.tar.gz
Already downloaded: /Library/Caches/Homebrew/rabbitmq-c--codegen-3.3.1.tar.gz
==> Verifying rabbitmq-c--codegen-3.3.1.tar.gz checksum
tar xf /Library/Caches/Homebrew/rabbitmq-c--codegen-3.3.1.tar.gz
==> autoreconf -i
glibtoolize: putting auxiliary files in `.'.
glibtoolize: copying file `./ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
glibtoolize: copying file `m4/libtool.m4'
glibtoolize: copying file `m4/ltoptions.m4'
glibtoolize: copying file `m4/ltsugar.m4'
glibtoolize: copying file `m4/ltversion.m4'
glibtoolize: copying file `m4/lt~obsolete.m4'
configure.ac:26: installing './compile'
configure.ac:30: installing './config.guess'
configure.ac:30: installing './config.sub'
configure.ac:23: installing './install-sh'
configure.ac:23: installing './missing'
Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
==> ./configure --prefix=/usr/local/Cellar/rabbitmq-c/HEAD --disable-dependency-tracking
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for style of include used by make... GNU
checking for gcc... clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether clang accepts -g... yes
checking for clang option to accept ISO C89... none needed
checking whether clang understands -c and -o together... yes
checking dependency style of clang... none
checking how to run the C preprocessor... clang -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking build system type... x86_64-apple-darwin14.0.0
checking host system type... x86_64-apple-darwin14.0.0
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/local/Library/ENV/4.3/sed
checking for fgrep... /usr/bin/grep -F
checking for ld used by clang... /Library/Developer/CommandLineTools/usr/bin/ld
checking if the linker (/Library/Developer/CommandLineTools/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm
checking the name lister (/usr/bin/nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 196608
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-apple-darwin14.0.0 file names to x86_64-apple-darwin14.0.0 format... func_convert_file_noop
checking how to convert x86_64-apple-darwin14.0.0 file names to toolchain format... func_convert_file_noop
checking for /Library/Developer/CommandLineTools/usr/bin/ld option to reload object files... -r
checking for objdump... no
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm output from clang object... ok
checking for sysroot... no
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if clang supports -fno-rtti -fno-exceptions... yes
checking for clang option to produce PIC... -fno-common -DPIC
checking if clang PIC flag -fno-common -DPIC works... yes
checking if clang static flag -static works... no
checking if clang supports -c -o file.o... yes
checking if clang supports -c -o file.o... (cached) yes
checking whether the clang linker (/Library/Developer/CommandLineTools/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin14.0.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for clang option to accept ISO C99... none needed
checking for pkg-config... /usr/local/opt/pkg-config/bin/pkg-config
checking pkg-config is at least version 0.17... yes
checking for inline... inline
checking if compiler accepts '-Wall'... yes
checking if compiler accepts '-Wextra'... yes
checking if compiler accepts '-pedantic'... yes
checking if compiler accepts '-Wstrict-prototypes'... yes
checking if compiler accepts '-Wcast-align'... yes
checking if compiler accepts '-fno-common'... yes
checking if compiler accepts '-fvisibility=hidden'... yes
checking if linker accepts '-no-undefined'... no
checking for library containing getaddrinfo... none required
checking for library containing socket... none required
checking for library containing clock_gettime... no
checking for htonll... no
checking for SSL... yes
checking for popt.h... no
checking for poptGetContext in -lpopt... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating librabbitmq.pc
config.status: creating Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands

rabbitmq-c build options:
    Host: x86_64-apple-darwin14.0.0
    Version: 0.5.2
    SSL/TLS: openssl
    Tools: no
    Documentation: no
    Examples: yes

==> make install
  CC       examples/examples_libutils_la-utils.lo
  CC       examples/amqp_bind.o
  CC       examples/amqp_consumer.o
  CC       examples/amqp_connect_timeout.o
  CC       examples/amqp_exchange_declare.o
  CC       examples/amqp_listen.o
  CC       examples/amqp_listenq.o
  CC       examples/amqp_producer.o
  CC       examples/amqp_rpc_sendstring_client.o
  CC       examples/amqp_sendstring.o
  CC       examples/amqp_unbind.o
  CC       examples/amqps_bind.o
  CC       examples/amqps_connect_timeout.o
  CC       examples/amqps_consumer.o
  CC       examples/amqps_exchange_declare.o
  CC       examples/amqps_listen.o
  CC       examples/amqps_listenq.o
  CC       examples/amqps_producer.o
  CC       examples/amqps_sendstring.o
  CC       examples/amqps_unbind.o
  CC       librabbitmq/librabbitmq_librabbitmq_la-amqp_api.lo
  CC       librabbitmq/librabbitmq_librabbitmq_la-amqp_connection.lo
  CC       librabbitmq/librabbitmq_librabbitmq_la-amqp_consumer.lo
  CC       librabbitmq/librabbitmq_librabbitmq_la-amqp_framing.lo
In file included from librabbitmq/amqp_api.c:41:
./librabbitmq/amqp_private.h:322:1: error: expected ')'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:322:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_api.c:41:
./librabbitmq/amqp_private.h:322:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
In file included from librabbitmq/amqp_api.c:41:
./librabbitmq/amqp_private.h:323:1: error: expected ')'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:323:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_api.c:41:
./librabbitmq/amqp_private.h:323:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
4 errors generated.
In file included from librabbitmq/amqp_connection.c:42:
./librabbitmq/amqp_private.h:322:1: error: expected ')'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:322:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_connection.c:42:
./librabbitmq/amqp_private.h:322:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
In file included from librabbitmq/amqp_connection.c:42:
./librabbitmq/amqp_private.h:323:1: error: expected ')'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:323:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_connection.c:42:
./librabbitmq/amqp_private.h:323:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
In file included from librabbitmq/amqp_consumer.c:31:
./librabbitmq/amqp_private.h:322:1: error: expected ')'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:322:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_consumer.c:31:
./librabbitmq/amqp_private.h:322:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
In file included from librabbitmq/amqp_consumer.c:31:
./librabbitmq/amqp_private.h:323:1: error: expected ')'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:323:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_consumer.c:31:
./librabbitmq/amqp_private.h:323:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
make: *** [librabbitmq/librabbitmq_librabbitmq_la-amqp_api.lo] Error 1
make: *** Waiting for unfinished jobs....
4 errors generated.
4 errors generated.
make: *** [librabbitmq/librabbitmq_librabbitmq_la-amqp_connection.lo] Error 1
make: *** [librabbitmq/librabbitmq_librabbitmq_la-amqp_consumer.lo] Error 1
In file included from librabbitmq/amqp_framing.c:41:
./librabbitmq/amqp_private.h:322:1: error: expected ')'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:322:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_framing.c:41:
./librabbitmq/amqp_private.h:322:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(hton)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
htonll
^
/usr/include/sys/_endian.h:141:25: note: expanded from macro 'htonll'
#define htonll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
In file included from librabbitmq/amqp_framing.c:41:
./librabbitmq/amqp_private.h:323:1: error: expected ')'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:30: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
                             ^
./librabbitmq/amqp_private.h:323:1: note: to match this '('
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:5: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
    ^
In file included from librabbitmq/amqp_framing.c:41:
./librabbitmq/amqp_private.h:323:1: error: definition of builtin function '__builtin_constant_p'
DECLARE_XTOXLL(ntoh)
^
./librabbitmq/amqp_private.h:288:26: note: expanded from macro 'DECLARE_XTOXLL'
  static inline uint64_t func##ll(uint64_t val)     \
                         ^
<scratch space>:141:1: note: expanded from here
ntohll
^
/usr/include/sys/_endian.h:140:25: note: expanded from macro 'ntohll'
#define ntohll(x)       __DARWIN_OSSwapInt64(x)
                        ^
/usr/include/libkern/_OSByteOrder.h:78:6: note: expanded from macro '__DARWIN_OSSwapInt64'
    (__builtin_constant_p(x) ? __DARWIN_OSSwapConstInt64(x) : _OSSwapInt64(x))
     ^
4 errors generated.
make: *** [librabbitmq/librabbitmq_librabbitmq_la-amqp_framing.lo] Error 1
==> Configuration
HOMEBREW_VERSION: 0.9.5
HEAD: d866e835c8473f8f92776cbf972f242854f03156
CPU: quad-core 64-bit haswell
OS X: 10.10-x86_64
Xcode: 6.0 => /Applications/Xcode6-Beta5.app/Contents/Developer
CLT: 6.0.0.0.1.1406926514
X11: N/A
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/ENV/pkgconfig/10.10
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/ENV/4.3:/usr/local/opt/pkg-config/bin:/usr/local/opt/autoconf/bin:/usr/local/opt/automake/bin:/usr/local/opt/libtool/bin:/usr/local/opt/git/bin:/usr/bin:/bin:/usr/sbin:/sbin

Error: rabbitmq-c HEAD did not build
Logs:
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/01.autoreconf
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/02.configure
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/02.configure.cc
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/03.make
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/03.make.cc
     /Users/dcousineau/Library/Logs/Homebrew/rabbitmq-c/config.log
```

Additional information

```
$ gcc --version
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 6.0 (clang-600.0.45.3) (based on LLVM 3.5svn)
Target: x86_64-apple-darwin14.0.0
Thread model: posix
```

--- a/configure.ac
+++ b/configure.ac
@@ -99,7 +99,26 @@ AC_SEARCH_LIBS([socket], [socket], [],
                              [AC_MSG_ERROR([cannot find socket library (library with socket symbol)])],
                              [-lnsl])])
 AC_SEARCH_LIBS([clock_gettime], [rt])
-AC_CHECK_FUNCS([htonll])
+AC_MSG_CHECKING([if htonll is defined])
+
+dnl # Check for htonll
+AC_LINK_IFELSE(
+  [AC_LANG_PROGRAM(
+     [[
+        #include <arpa/inet.h>
+     ]],
+     [[
+       return htonll(0);
+     ]]
+  )],
+  [
+    AC_MSG_RESULT(yes)
+    AC_DEFINE(HAVE_HTONLL, 1, [Define to 1 if the function (or macro) htonll exists.])
+  ],
+  [
+    AC_MSG_RESULT(no)
+  ]
+)
 
 # Configure SSL/TLS
 AC_ARG_WITH([ssl],

